pcdsdevices/lodcm.py

```diff
@@ -60,6 +65,98 @@ def __init__(self, prefix, *args, **kwargs):
         super().__init__(prefix, *args, **kwargs)
 
 
+class CHI1(InOutRecordPositioner):
+    states_list = ['C', 'Si']
+    in_states = ['C', 'Si']
+    out_states = []
+
+
+class CHI2(InOutRecordPositioner):
+    states_list = ['C', 'Si']
+    in_states = ['C', 'Si']
+    out_states = []
+
+
+class H2N(InOutRecordPositioner):
+    states_list = ['C', 'Si']
+    in_states = ['C', 'Si']
+    out_states = []
+
+
+class Y1(InOutRecordPositioner):
+    states_list = ['C', 'Si']
+    in_states = ['C', 'Si']
+    out_states = []
+
+
+class Y2(InOutRecordPositioner):
+    states_list = ['C', 'Si']
+    in_states = ['C', 'Si']
+    out_states = []
+
+
+LODCM_MOTORS = {
+    # CRYSTAL TOWER ONE
+    'z1': {'prefix': 'XPP:MON:MMS:04', 'description': 'LOM Xtal1 Z',
+           'motor': Motor('XPP:MON:MMS:04', name='z1')},
```

Comment:
The calls to the factory function `Motor()` are instantiating ophyd devices that talk to PVs on import - this will have to be removed/refactored.

Reply:
ohh no, i see - thanks @Ken, i'll remove it because I don't need it anyway